common/lib/dependabot/pull_request_updater/azure.rb

```diff
@@ -0,0 +1,113 @@
+# frozen_string_literal: true
+
+require "dependabot/clients/azure"
+require "uuid"
```

Comment:
You can use this module from the standard library instead:

```suggestion
require "securerandom"
```

Reply:
Cool Thanks for the suggestion!